### PR TITLE
ADCM-3091 raise "400 INVALID_DATA" on policy update with no "id" key

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

**Review by SofiaTal — DISMISSED** [review]

File: python/rbac/utils.py [hunk 1]
```diff
@@ -15,11 +15,16 @@
...
+
 
 class BaseRelatedSerializer(serializers.Serializer):
     def to_internal_value(self, data):
         data = super().to_internal_value(data)
-        return data['id']
+        try:
+            return data["id"]
+        except KeyError as e:
+            raise AdcmEx("INVALID_DATA", f"Field {e} is required") from e
```
Comment (a-alferov) [inline python/rbac/utils.py:27]:
```suggestion
        if "id" not in data:
            raise serializers.ValidationError("This field may not be empty.")
        return data.get("id")
```
`is required` invalid error for parent class

File: python/cm/errors.py [hunk 2]
```diff
@@ -177,6 +177,11 @@
         rfs.HTTP_400_BAD_REQUEST,
         ERR,
     ),
+    "INVALID_DATA": (
```
Comment (a-alferov) [inline python/cm/errors.py:180]: it's too much

File: python/rbac/tests/tests_api/test_policy_unittests.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
```
Comment (a-alferov) [inline python/rbac/tests/tests_api/test_policy_unittests.py:15]: please use Django client instead of DRF client

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/cm/errors.py (hunk 2 above)
Comment (DanBalalan) [reply python/cm/errors.py]: removed

File: python/rbac/tests/tests_api/test_policy_unittests.py (hunk 3 above)
Comment (DanBalalan) [reply python/rbac/tests/tests_api/test_policy_unittests.py]: done

File: python/rbac/utils.py (hunk 1 above)
Comment (DanBalalan) [reply python/rbac/utils.py]: done

**Review by a-alferov — APPROVED** [review]

**Pull request merged**